helm-charts/chatqna/templates/horizontalPorAutoscaler.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.horizontalPodAutoscaler.enabled }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "chatqna.fullname" . }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "chatqna.fullname" . }}
+  minReplicas: 1
+  maxReplicas: 6
```

Comment:
It's better if user can specify suitable `maxReplicas` value for each HPA controlled deployment in `values.yaml`.

(Especially as none of these deployments specifies their resource usage, nor anti-/affinity rules, so that k8s could avoid scheduling too many instances on the same nodes.)

Reply:
Done.